### DISCUSSION TITLE
github student pack offer

### DISCUSSION THREAD

**Discussion by mevivekkaushik** [community/community#121208] [GitHub Education]
### Select Topic Area

Question

### Body

i have submiited many times different proof of github but he

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @vivekkaushik004 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by mevivekkaushik** [reply]
Can you give your contact no

**Reply by samus-aran** [reply]
Hi @vivekkaushik004 

As awesome as we are we don't give out personalised support in any capacity. We'll aid our members the best we can within these discussions. 🙂

**Comment by samus-aran** [comment]
Hi @vivekkaushik004 (again)

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
